src/neo/VM/Helper.cs

```diff
@@ -272,6 +272,12 @@ private static ContractParameter ToParameter(StackItem item, List<(StackItem, Co
                         Type = ContractParameterType.InteropInterface
                     };
                     break;
+                case Null _:
+                    parameter = new ContractParameter
+                    {
+                        Type = ContractParameterType.Any
```

Comment:
Is not better to create a new one, Null?

Reply:
I think `null` is no a type.